Assets/Treasure/TDK/Runtime/Connect/UI/TDKEmbeddedWalletUI.cs

```diff
@@ -10,9 +10,6 @@ namespace Treasure
 {
     public class TDKEmbeddedWalletUI : InAppWalletUI
     {
-        [Space]
-        [Tooltip("Invoked when the user completes OTP process.")]
-        public UnityEvent OnEmailOTPVerificationSuccess;
```

Comment:
I removed this because
1. I didnt see it being used anywhere
2. It was being emitted when any OTP is submitted, even invalid ones (it was in a `finally` clause)

Not sure if this was exposed here as a feature for TDK users to add listeners for this case or if it was used for something else that was removed at some point. To me it seemed a bit too specific, so i removed it, but let me know if perhaps i should re-add it and change where its emitted

Reply:
@lefarchi at one point we needed to expose hooks for partners to be aware of the connect status. I can't recall if it was off this specific UnityEvent or elsewhere 🤔

Reply:
There is a `OnConnected` event on `TDK.Connect.cs`, maybe its that one? We now have several social logins, so if partners use this OTP hook then it will only work for 1 of the connect paths